articles_test.go

```diff
@@ -0,0 +1,205 @@
+package goshopify
+
+import (
+	"context"
+	"fmt"
+	"github.com/jarcoal/httpmock"
```

Comment:
nit: break out the 3rd party import in a separate section

Reply:
thanks! done.